client/host_queue.go

```diff
@@ -177,16 +183,20 @@ func (q *queue) rotateOpsWithLock() []op {
 
 func (q *queue) drain() {
 	var (
-		currWriteOpsByNamespace      = make(map[ident.Hash][]op)
-		currBatchElementsByNamespace = make(map[ident.Hash][]*rpc.WriteBatchRawRequestElement)
-		writeBatchSize               = q.opts.WriteBatchSize()
+		currWriteOpsByNamespace            = make(map[ident.Hash][]op)
+		currBatchElementsByNamespace       = make(map[ident.Hash][]*rpc.WriteBatchRawRequestElement)
+		currTaggedWriteOpsByNamespace      = make(map[ident.Hash][]op)
+		currTaggedBatchElementsByNamespace = make(map[ident.Hash][]*rpc.WriteTaggedBatchRawRequestElement)
```

Comment:
Arg... I really need to get the generalized map ready.

Reply:
:)